test.py

```diff
@@ -63,6 +64,10 @@ class palette:
     diff_mark = create_formatter(colorama.Fore.MAGENTA)
     warn = create_formatter(colorama.Fore.YELLOW)
     crit = create_formatter(colorama.Fore.RED, colorama.Style.BRIGHT)
+    ansi_escape = re.compile(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])')
```

Comment:
I guess you tested this function, but I can't readit and wonder how it matches an ANSI color change sequence like `ESC[01;34m`.

Reply:
Yes, I did test it, it removes the escape sequences added by colorama. I think you are mixing up ESC as three letters - E - S - C and ESC as a key on the terminal keyboard with an assigned ANSI code which is 1B, that is 27 in decimal. The escape sequence starts with ESC control code, not ESC letters.